va/tlsalpn.go

```diff
@@ -189,14 +200,27 @@ func (va *ValidationAuthorityImpl) validateTLSALPN01(ctx context.Context, identi
 
 	leafCert := certs[0]
```

Comment:
I think length checking here as well?

Reply:
the function which generates this value throws an error if the length of certs is 0